app/services/identity/transfer.rb

```diff
@@ -13,6 +13,8 @@ def call
         transfer_induction_coordinator_profile!
         transfer_get_an_identity_id!
         create_participant_id_change!
+      rescue ActiveRecord::Rollback
```

Comment:
discussed in person: we'll be using: `ActiveRecord::Base.transaction(requires_new: true)` instead to avoid raising errors due to the usage of this service across the app